Workade/Managers/NetworkManager.swift

```diff
@@ -46,6 +46,22 @@ final class NetworkManager {
             fatalError("Failed json parsing")
         }
     }
+    
+    func fetchCheckListTemplateData<T: Codable>() async throws -> T {
```

Comment:
```suggestion
    func requestCheckListTemplateData<T: Codable>() async throws -> T {
```
[P2] 문맥상 항상 갖고 올 수 있는 데이터는 아니니 fetch보다는 request가 맞다고 생각됩니다!

Reply:
아하 네! 그렇게 적용하겠습니다!